lib/active_merchant/billing/gateways/card_connect.rb

```diff
@@ -241,10 +241,12 @@ def add_additional_data(post, options)
         post[:userfields] = options[:user_fields] if options[:user_fields]
       end
 
-      def add_3DS(post, options)
-        post[:secureflag] = options[:secure_flag] if options[:secure_flag]
-        post[:securevalue] = options[:secure_value] if options[:secure_value]
-        post[:securexid] = options[:secure_xid] if options[:secure_xid]
+      def add_three_ds(post, options)
```

Comment:
🛑   Important question:

I see here that you are removing 3DS v1 fields, does this change means that card connect is dropping support for 3DS v1 and just supports now 3DS v2 ?

Checking remote test 3DS v1 seems still working ...

Reply:
I thing they had not support for [3ds1]( https://docs.spreedly.com/payment-gateways/card-connect/ ) , instead it seems, this was an old implementation for 3ds2 .

Reply:
Hi @curiousepic looking into the Documentation [here](https://support.cardpointe.com/compliance/3-d-secure-2-0#if-i-already-use-3-d-secure-1-0-do-i-need-to-update-to-2-0 ), it seems they don't have support for 3ds v1 since 2020. 
Looks like they had an deprecated implementation in AM independently to Spreedly core, I mean in core they didn't have enable this feature. 
what do you thing?.

Reply:
I see, this implementation looks good according to those docs 👍